src/core/http-request/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpMethod, HttpBody, HttpHeaders, HttpMethods } from "../http";
+import { stringBody, HttpRequestImpl } from "../../node";
+
+export type RequestParams = {
+  url: string;
+  method?: HttpMethod; // Method can also be optional in this function
+  body?: HttpBody | string;
+  headers?: HttpHeaders;
+};
+
+/**
+ * Instantiates an HttpResponse based on the provided data
+ * @param obj Request representative object. Only url is mandatory. The default HttpMethod is GET.
+ */
+export function req({
```

Comment:
Are we going to keep this function with a single parameter as an object or several parameters?

Reply:
I think we have decided to keep it like that.